br/pkg/task/backup_raw.go

```diff
@@ -49,6 +51,10 @@ func DefineRawBackupFlags(command *cobra.Command) {
 	command.Flags().Bool(flagRemoveSchedulers, false,
 		"disable the balance, shuffle and region-merge schedulers in PD to speed up backup.")
 
+	command.Flags().Duration(flagSafeInterval, utils.DefaultBRSafeInterval,
+		"The interval between backup-ts and current tso.")
+	command.Flags().Int64(flagGCTTL, utils.DefaultBRGCSafePointTTL, "the TTL (in seconds) that PD holds for BR's GC safepoint")
```

Comment:
Suggest to be a `duration` argument.
TTL would be expected to be e.g, 30 minutes, or 3 days. It is too big to be in secondes.

```suggestion
    command.Flags().Int64(flagGCTTL, utils.DefaultBRGCSafePointTTL, "the TTL of BR's GC safepoint")
```

Reply:
done